phpcs.ruleset.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset name="GiveWP Coding Standards">
+	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Core/ruleset.xml -->
+
+	<!-- Set a description for this ruleset. -->
+	<description>We love clean code for GiveWP. This will let us enforce that easily.</description>
+
+	<!-- Exclude directories we don't want checked. -->
+	<exclude-pattern>node_modules/*</exclude-pattern>
+	<exclude-pattern>vendor/*</exclude-pattern>
+	<exclude-pattern>sample-data/*</exclude-pattern>
```

Comment:
@alaca can we remove unnecessary directories from this list

Reply:
Sample data? Yes.

Reply:
Is this resolved, @alaca?